class/defaults.yml

```diff
@@ -3,3 +3,19 @@ parameters:
     =_metadata:
       multi_tenant: true
     namespace: syn-espejote
+
+    manifestVersion: main
+    kustomizeInput:
```

Comment:
We should have an easy way to override resources as they might be too small soon enough with a few bigger patches.